src/main/java/org/zalando/planb/provider/AuthorizeController.java

```diff
@@ -49,23 +99,27 @@
     @Autowired
     private CassandraAuthorizationCodeService cassandraAuthorizationCodeService;
 
+    @Autowired
+    private CassandraConsentService cassandraConsentService;
+
     /**
-     * Authorization Request, see http://tools.ietf.org/html/rfc6749#section-4.1.1
+     * Authorization Request, see http://tools.ietf.org/html/rfc6749#section-4.1.1.
      */
-    @RequestMapping(value = "/oauth2/authorize")
-    String showAuthorizationForm(@RequestParam(value = "realm") Optional<String> realmNameParam,
-                                 @RequestParam(value = "response_type", required = true) String responseType,
-                                 @RequestParam(value = "client_id", required = true) String clientId,
-                                 @RequestParam(value = "scope") Optional<String> scope,
-                                 @RequestParam(value = "redirect_uri") Optional<URI> redirectUriParam,
-                                 @RequestParam(value = "state") Optional<String> state,
-                                 @RequestParam(value = "error") Optional<String> error,
-                                 @RequestHeader(value = "Host") Optional<String> hostHeader,
-                                 Model model) {
+    @RequestMapping
+    String showAuthorizationForm(@RequestParam(value = REALM) final Optional<String> realmNameParam,
+            @RequestParam(value = RESPONSE_TYPE_SNAKECASE, required = true) final String responseType,
```

Comment:
I personally would not extract all those param names to constants, because it is harder to read (as Victor already mentioned before). It also won't improve the performance of the application, because annotations and all of their parameter values are constant anyway.